src/Adapter/Order/CommandHandler/AddOrderFromBackOfficeHandler.php

```diff
@@ -71,15 +71,14 @@ public function __construct(ContextStateManager $contextStateManager)
      */
     public function handle(AddOrderFromBackOfficeCommand $command)
     {
-        /** @var PaymentModule $paymentModule */
         $paymentModule = !Configuration::get('PS_CATALOG_MODE') ?
             Module::getInstanceByName($command->getPaymentModuleName()) :
             new BoOrderCore();
 
         if (false === $paymentModule) {
             throw new OrderException(sprintf('Payment method "%s" does not exist.', $paymentModule));
         }
-
+        /** @var PaymentModule $paymentModule */
```

Comment:
Maybe add an empty line here to better understand that `$paymentModule` has nothing to do with `$cart`?
```suggestion
        /** @var PaymentModule $paymentModule */
        
```

Reply:
Done